test.py

```diff
@@ -228,7 +227,55 @@ def test_difficult_words():
     textstat.set_lang("en_US")
     result = textstat.difficult_words(long_test)
 
-    assert result == 49
+    assert set(result) == set(['exist',
```

Comment:
Could we add a new test for `difficult_words_list()`, and use `short_test` instead?

```python
assert textstat.difficult_words_list(short_test) == ["sunglasses"] # I think?
```

Reply:
Done!